### ISSUE TITLE
How to change Crepe configurations to TEST Dataset on Trained Model

### ISSUE THREAD

**Issue by awaiskhan2007** [zhangxiangxiao/Crepe#18]
I was wondering, how to use already trained model for validating my dataset; Can you please highlight all steps how to manage such change in crepe because -resume is not fulfilling my requirement;

Lastly, If I want to change error function to predict only one label, can you shed some light on that as well?

Regards,
Khan Awais

**Comment by awaiskhan2007** [comment]
Actually, because -resume, it every time starts training model which I don't want; can you please also provide some details about -resume in readme file also if possible;

**Comment by ch3njust1n** [comment]
I had a similar question to your first question. According to Zhang, you need to dig into the code and place print line statements in main.lua for test.decision[i]. These are the predicted labels. I was considering just using the config.lua and creating a script to do the feedforward on new unlabeled data. Haven't gotten around to doing that yet, so I'm not sure about all the details.

As for your second question about predicting only one label,  you can find and change code in main.lua for controlling what gets graphed. I haven't tried changing that, but if you dig around the code, you'll probably figure it out.

Hope that helps.

**Comment by awaiskhan2007** [comment]
Thanks Justin for your fruitful feedback; but my question is bit different, is there any way to omit training step, because i already had trained model. I simply want to use that model. Can I comment training code in main.lua? will it plot proper graph for me? can simply tell me how resume works etc?

Apologies for asking too much

**Comment by ch3njust1n** [comment]
I'm not familiar enough with the code to say which pieces to comment out, but according to Zhang, you can do something like: model = torch.load("sequential_EPOCHES_TIME.t7b") and then I'm assuming you can just write the quantization and feedfoward code along with his config.lua to use your trained model. I plan on doing this as well. Let me know if you figure it out.

**Comment by zhangxiangxiao** [comment]
You can comment out code that runs training in main.lua.
